packages/p2p-v2/src/pages/my-profile/screens/MyProfileBalance/MyProfileBalance.tsx

```diff
@@ -1,74 +1,97 @@
+                            <Text size='sm'>Buy</Text>
+                            <div className='p2p-v2-my-profile-balance__item-limits'>
+                                <div>
+                                    <Text color='less-prominent' size='sm'>
+                                        Daily limit
+                                    </Text>
+                                    <Text className='p2p-v2-my-profile-balance__label' size='sm' weight='bold'>
+                                        {advertiserInfo?.daily_buy_limit} USD
+                                    </Text>
+                                </div>
+                                <div>
+                                    <Text color='less-prominent' size='sm'>
+                                        Available
+                                    </Text>
+                                    <Text className='p2p-v2-my-profile-balance__label' size='sm' weight='bold'>
+                                        {numberToCurrencyText(advertiserInfo.dailyAvailableBuyLimit)} USD
+                                    </Text>
+                                </div>
+                            </div>
                         </div>
-                        <div>
-                            <Text color='less-prominent' size='sm'>
-                                Available
-                            </Text>
-                            <Text className='p2p-v2-my-profile-balance__label' size='sm' weight='bold'>
-                                {numberToCurrencyText(advertiserInfo.dailyAvailableBuyLimit)} USD
-                            </Text>
+                        <div className='p2p-v2-my-profile-balance__item'>
+                            <Text size='sm'>Sell</Text>
+                            <div className='p2p-v2-my-profile-balance__item-limits'>
+                                <div>
+                                    <Text color='less-prominent' size='sm'>
+                                        Daily limit
+                                    </Text>
+                                    <Text className='p2p-v2-my-profile-balance__label' size='sm' weight='bold'>
+                                        {advertiserInfo?.daily_sell_limit} USD
+                                    </Text>
+                                </div>
+                                <div>
+                                    <Text color='less-prominent' size='sm'>
+                                        Available
+                                    </Text>
+                                    <Text className='p2p-v2-my-profile-balance__label' size='sm' weight='bold'>
+                                        {numberToCurrencyText(advertiserInfo.dailyAvailableSellLimit)} USD
+                                    </Text>
+                                </div>
+                            </div>
                         </div>
                     </div>
```

Comment:
could you put the string values into an array and then map them? cause the stylings are similar here